src/app/components/atoms/pokemonIndexDetail/index.tsx

```
@@ -4,51 +4,51 @@ import TypeList from "../typeList";
+			className={clsx(
+				"flex flex-col items-center p-4 bg-white rounded-lg shadow-md transition-all duration-300",
+				"hover:shadow-xl hover:scale-105",
+				"cursor-pointer",
+				"border border-gray-200",
+				hoveredPokemon === pokemon.id && "bg-gray-100",
+				"h-full",
+			)}
+			onClick={() => handleModelOpen && handleModelOpen(pokemon)}
+			onMouseEnter={() => setHoveredPokemon && setHoveredPokemon(pokemon.id)}
+			onMouseLeave={() => setHoveredPokemon && setHoveredPokemon(null)}
+		>
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)